src/Core/src/Platform/iOS/PageExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	public static class PageExtensions
```

Comment:
Should this be internal, so we don't add any new public API?

Reply:
It's probably too late. Changed. Could you review it again?. Thnx!